src/main/webapp/scripts/mapscript.js

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 Google LLC
```

Comment:
nit: it should be fine to omit `script` from the file name as long as it's clearly a JavaScript file. Otherwise please use a hyphen separator between the two words
https://google.github.io/styleguide/jsguide.html#file-name

Reply:
Ah thank you!